src/Illuminate/Container/Container.php

```diff
@@ -566,11 +566,11 @@ protected function getReboundCallbacks($abstract)
     /**
      * Wrap the given closure such that its dependencies will be injected when executed.
      *
-     * @param  \Closure  $callback
+     * @param  callable|string  $callback
      * @param  array  $parameters
      * @return \Closure
      */
-    public function wrap(Closure $callback, array $parameters = [])
+    public function wrap($callback, array $parameters = [])
```

Comment:
This is a breaking change unfortunately.

Reply:
The method signature has changed.

Reply:
Ok. Should I send PR to the 7.x Laravel version or something else? Can you tell me, are these changes useful in general?

Reply:
Taylor generally reviews the prs but I don't think this will pass no.

Reply:
I realized your comment about BC. 

Is anyone else can check this PR? Is this PR may be useful for version 7.x or 8.x?